### PR TITLE
Feature/mat 7522 qicore builder

### PR DESCRIPTION
**Pull request by nmorasb**
## MADiE PR

Jira Ticket: [MAT-7522](https://jira.cms.gov/browse/MAT-7522)
(Optional) Related Tickets:

### Summary

Backend code relevant to running these frontend changes is available here: 
[fhir-server](https://github.com/nmorasb/fhir-server)

### All Submissions
* [x] This PR has the JIRA linked.
* [ ] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -94,7 +94,8 @@
     "@mui/lab": "^5.0.0-alpha.82",
     "@mui/material": "^5.6.2",
     "@mui/styles": "^5.8.0",
-    "@mui/x-date-pickers": "^6.6.0",
+    "@mui/x-data-grid": "^6.18.6",
+    "@mui/x-date-pickers": "^6.18.7",
```
Comment (nmorasb) [inline package.json]: These were only added for resource table in PoC. Once we play the resource table tanstack story (probably hasn't even been written yet) these dependencies can be removed

**Review by sb-cecilialiu — APPROVED** [review]
